tests/run-make/print-native-static-libs/rmake.rs

```diff
@@ -0,0 +1,56 @@
+//! This checks the output of `--print=native-static-libs`
+
+//@ ignore-cross-compile
+//@ ignore-wasm
+
+extern crate run_make_support;
+
+use std::io::BufRead;
+
+use run_make_support::rustc;
+
+fn main() {
+    // build supporting crate
+    rustc()
+        .input("bar.rs")
+        .arg("--crate-type")
+        .arg("rlib")
+        .arg("-lbar_cli")
+        .run();
+
+    // build main crate as staticlib
+    let output = rustc()
+        .input("foo.rs")
+        .arg("--crate-type")
+        .arg("staticlib")
+        .arg("-lfoo_cli")
+        .arg("-lfoo_cli") // 2nd time
```

Comment:
Probably worth adding a `l("foo_cli")` helper on `Rustc`, but no strong opinions on this specifically.

Reply:
If you don't mind I would prefer keeping the full argument here since we specifically try to match on it below.

Reply:
That's fine then